### ISSUE TITLE
Ghost core: make comments accessible

### ISSUE THREAD

**Issue by peterfremlin** [peterfremlin/Disability-Debrief#13]
Comment threading not visible on screenreader
https://forum.ghost.org/t/accessibility-native-comments-threading-not-visible-to-screenreader/32312
